### PR TITLE
Polished README.md file to increase readability and key focus points

### PR DESCRIPTION
**Pull request by mzajkowski**
### Description

I got lost... I got lost in Umbraco's README not once, but multiple times. Then I checked some of the "best" repos on Github (referring to Octoverse rankings and popularity of some of the tools) and then I realised that Umbraco's readme file is not about what's the most important HERE (on Github) - SOFTWARE.

I've removed two links to the introduction video which was probably rarely watched from here + containing the old branding (#legacyorange) + I've organised elements into the clean "agenda" with anchor links and external links to DOCUMENTATION of the project and some more useful details related with specific areas of CONTRIBUTION.

I think (and hope) it might be a great starting point to improve it even further and make this initial page "working" for people who are visiting it the most - devs.

I'm really, really looking forward for your feedback...

### CODE REVIEW

**Comment by emmaburstow** [comment]
Hi Marcin!

Thanks for the Pr! We will take a look at the changes and let you know if we need anything from you.

Em

File: .github/README.md [hunk 1]
```diff
@@ -1,50 +1,50 @@
...
-## Contribute to Umbraco
+Our friendly community is available 24/7 at the community hub we call ["Our Umbraco"](https://our.umbraco.com/). Our Umbraco features forums for questions and answers, documentation, downloadable plugins for Umbraco, and a rich collection of community resources.
 
-Umbraco is contribution-focused and community-driven. If you want to contribute back to Umbraco, please check out our [guide to contributing](CONTRIBUTING.md).
+Besides "Our", we all support each other also via:
 
-## Found a bug?
+- Twitter: [Umbraco HQ](https://twitter.com/umbraco), [Release Updates](https://twitter.com/umbracoproject), [#umbraco](https://twitter.com/hashtag/umbraco)
+- Gitter: https://gitter.im/umbraco/Umbraco-CMS
+- Slack: http://umbracians.chat/
```
Comment (abjerner) [inline .github/README.md]: Hi @mzajkowski,

The URL for Slack is no longer working, and I don't think it has been working for some months. The URL just results in a 403 on Azure. Could you remove the link? From what I've seen on Slack, there isn't a new URL to be used instead.

File: .github/README.md [hunk 2]
```diff
@@ -1,50 +1,50 @@
...
 
-Another way you can contribute to Umbraco is by providing issue reports. For information on how to submit an issue report, refer to our [online guide for reporting issues](https://github.com/umbraco/Umbraco-CMS/blob/dev-v7/.github/CONTRIBUTING.md).
+## License
 
-You can comment and report issues on the [github issue tracker](https://github.com/umbraco/Umbraco-CMS/issues).
-Since [September 2018](https://umbraco.com/blog/a-second-take-on-umbraco-issue-tracker-hello-github-issues/), the old issue tracker is in read-only mode, but can still be found at [http://issues.umbraco.org](http://issues.umbraco.org).
+Umbraco CMS is and always will be [MIT licensed](LICENSE.md) open source software.
```
Comment (abjerner) [inline .github/README.md]: I can't really tell whether this is one of the things that will work when this PR is merged into `dev-v7`, but I think the link to the license file is broken (both the badge and the link at the bottom).

As `README.md` is located inside the magic `.github` folder, GitHub will generate a link relative to this folder - but `LICENSE.md` is located at the root of the repository.

I've tested this be accessing your own fork here: https://github.com/mzajkowski/Umbraco-CMS/tree/temp-polished-readme

Comment (mzajkowski) [reply .github/README.md]: You're right @abjerner! I've missed it. Fixing both issues now...

**Comment by mzajkowski** [comment]
@emmaburstow @abjerner I've updated both suggestions and tested it on my fork (URLs). Seems to work fine now! Good spots. Waiting for your further actions / feedback.

**Comment by nul800sebastiaan** [comment]
Thanks @mzajkowski - this is a bit sensitive as someone in marketing has worked hard on the current content, so I'll have to discuss the changes at HQ first.

**Comment by abjerner** [comment]
@nul800sebastiaan any progress on this one?

**Comment by mzajkowski** [comment]
Maybe worth to re-visit it before the "fresh" start on Tuesday? Just noticed some broken links in the current readme too (e.g. to the v7 dev branch which is now linking to self / current branch).

+ I've resolved the conflicts and was just going to ask: what it shows / tells to the repo visitor that Pull Reminders are used? Is anyone able to configure it to their slack? How it works?

**Comment by nul800sebastiaan** [comment]
Thanks again @mzajkowski - I have taken some of this PR as inspiration but because it didn't want to merge properly at all, I'll close this one. Your efforts have been recognized however and we're sending you a virtual #h5yr!

**Pull request closed without merging**